packages/socket/package.json

```diff
@@ -5,23 +5,30 @@
   "main": "index.js",
   "scripts": {
     "clean-deps": "rm -rf node_modules",
+    "postinstall": "patch-package",
```

Comment:
Should `patch-package` now be moved from the root to the individual packages that use/require it? It was only added to the root to make it easier to work with, but it seems the opposite is actually true. What do you think about adding `patch-package` as a dev dependency to the packages that use it and remove it from the root `package.json`?

Reply:
I believe it needs to stay a root dependency - during binary-build, only production dependencies are installed, so patch-package wouldn't be installed at `postinstall`.